libs/wire-api/src/Wire/API/Event/Conversation.hs

```diff
@@ -461,13 +498,14 @@ instance ToJSON MemberUpdateData where
 
 instance FromJSON MemberUpdateData where
   parseJSON = withObject "member-update event data" $ \m ->
-    MemberUpdateData <$> m .:? "target"
-      -- NOTE: This is really not a maybe and should
+    MemberUpdateData
+      -- NOTE: This (target) is really not a maybe and should
       --       be made compulsory 28 days after the next
       --       release to prod to guaratee that no events
       --       out there do not contain id.
```

Comment:
I think I'll just change the comment to say that it can be removed immediately. I'll also link the ticket: https://github.com/zinfra/backend-issues/issues/1309